### PR TITLE
DDF for Bosch Smoke Alarm II (#9)

### PR DESCRIPTION
**Pull request by Finne75**
* Create bsd-2-smoke-alarm.js

* Rename bsd-2-smoke-alarm.js to bsd-2-smoke-alarm.json

### CODE REVIEW

**Comment by Finne75** [comment]
Hu? Validation error for DDF code provided by the deCONZ GUI DDF editor?

File: devices/bosch/bsd-2-smoke-alarm.json [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/battery",
+          "parse": {
+            "at": "0x0021",
+            "cl": "0x0001",
+            "ep": 1,
+            "eval": "Item.val = Attr.val / 2",
+            "fn": "zcl"
```
Comment (SwoopX) [inline devices/bosch/bsd-2-smoke-alarm.json:63]: This line's why the validator complains. We've recently decided to rename the function, it now should be `zcl:attr`

File: devices/bosch/bsd-2-smoke-alarm.json [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+          },
+          "default": 0
```
Comment (SwoopX) [inline devices/bosch/bsd-2-smoke-alarm.json:65]: You can remove the default here

**Review by SwoopX — CHANGES_REQUESTED** [review]

**Comment by SwoopX** [comment]
Really good DDF. One question though: does the device respond to arbitrary read requests without waking it up?

**Comment by Finne75** [comment]
> Really good DDF. One question though: does the device respond to arbitrary read requests without waking it up?

The device responds to read requests, e.g. to read a cluster via GUI. It does so at any time as far as I noticed.

**Comment by manup** [comment]
@SwoopX PR looks good to me, ready to merge?

**Review by SwoopX — APPROVED** [review]

**Comment by SwoopX** [comment]
@manup Yeah, all looking good. You could remove the `"awake": true` parts, as the device seems to be a light sleeper, but that's just a blemish 😉

**Pull request merged**